myfirstfile.js

```diff
@@ -7,4 +7,11 @@ let d = 10;
 
 const arr = [];
 
-console.log('first commit');
\ No newline at end of file
+console.log('first commit');
+
+function sumNums(a, b, c) {
+    return (a + b + c);
+}
+
+let result = sumNums(a, c, d);
```

Comment:
what are the values for `a`, `b` and `c`?

Reply:
Values for a, c, d were assigned above in lines 4, 5, 6.